Lundii/Banka
# #165125004 Download necessary packages

#### What does this PR do?
Downloads necessary packages used for the project
#### Description of Task to be completed?
Not available
#### How should this be manually tested?
See the package.json file on the root directory to see the development dependencies
#### Any background context you want to provide?
No
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/165125004
#### Screenshots (if appropriate)
No
#### Questions:
No
